src/cryptonote_core/service_node_list.cpp

```diff
@@ -563,33 +619,60 @@ namespace service_nodes
     }
   }
 
-  void service_node_list::update_swarms(uint64_t height) {
```

Comment:
Why did this method need to be manually inlined?  We should be moving *away* from massive functions towards smaller, more focused ones where possible and easily logically separable.